content/en/docs/Endpoints/getplayqueuebyindex.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "getPlayQueueByIndex"
+linkTitle: "getPlayQueueByIndex"
+categories:
+  - Bookmarks
+OpenSubsonic:
+  - Extension
+description: >
+  Returns the state of the play queue for this user.
+---
+
+`http://your-server/rest/getPlayQueueByIndex` Since [1.12.0](../../subsonic-versions)
+
+Returns the state of the play queue for this user (as set by `savePlayQueue` or `savePlayQueueByIndex`). This includes the tracks in the play queue, the currently playing track index (1-based), and the position within this track. Typically used to allow a user to move between different clients/apps while retaining the same play queue (for instance when listening to an audio book).
```

Comment:
I would suggest that this be 0-based because elsewhere in the API (e.g. `updatePlaylist`, there is already the concept of 0-based indexes)